modules/ossm-rn-new-features.adoc

```diff
@@ -33,9 +33,13 @@ Module included in the following assemblies:
 |1.24.1
 
 |Kiali
-|1.36.5
+|1.36.7
 |===
 
+== New features {ProductName} 2.1.1
+
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
```

Comment:
I think we also want to add a sentence here to mention that this release also adds the ability to disable the automatic creation of network policies.